### ISSUE TITLE
Bitfinex

### ISSUE THREAD

**Issue by gelinger777** [FlorianWilhelm/zipline-poloniex#7]
Hi there. This plugin looks great ... Need time to have deeper look. But a fast question. Which files need to be rewritten so that we could have one with bitfinex instead of poloniex. ? 
Do you think it would be hard to implement 
https://github.com/ccxt/ccxt ?

**Comment by FlorianWilhelm** [comment]
Hi @gelinger777, cool idea including [ccxt](https://github.com/ccxt/ccxt). I think basically (without taking a deeper look at ccxt), one would need to replace `api.py` with a proper interface to ccxt. Overall it should not be that difficult, do you want to contribute a PR?

**Comment by gelinger777** [comment]
we need some time thought but maybe yes. Only adapting api.py would be enought you think?

**Comment by FlorianWilhelm** [comment]
Yes, I assume that making instead of `api.py` a wrapper around ccxt should be fine but I haven't looked into it.

**Comment by gelinger777** [comment]
I have forked your project, so I will play a bit. Lets see.  I will come back to you later on if I have questions.

**Comment by avoll** [comment]
@gelinger777, I was just thinking along the same lines, although not for bitfinex, but for any ccxt supported exchange. We could collaborate potentially?
@FlorianWilhelm, with your permissions, I'd like to fork and refactor for this.

**Comment by FlorianWilhelm** [comment]
@avoll, cool, go for it! Looking forward to your contribution.

**Comment by gelinger777** [comment]
@avoll  yes we can collaborate. would be interesting. I will try to find some time next week.

**Comment by mayzenator** [comment]
Hi! Has this been implemented yet?

**Comment by avoll** [comment]
Alas, not by me. I switched to a different solution soon after posting my
interest.

It would not be much trouble to do it in ccxt, perhaps even directly
implementing bundle interface.
